scripts/cli/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptsArray=(
+    "schema_test"
+    "0_create_identities"
+    "1_poly_transfer"
+    "2_key_management"
+    "3_auth_join_did"
+    "4_permission_management"
+    "5_claim_management"
+    "6_create_assets"
+    "7_create_claim_compliance"
+    "8_asset_transfer"
+    "10_bridge_transfer"
+    "11_governance_management"
+    "12_A_settlement"
+    "12_B_settlement"
```

Comment:
Instead of mentioning all the files, I think it is good to use regex base on numbers so you don't need to append this every time when you add a new file in the test.

Reply:
I made a change that may reflect what you are mentioning where you no longer have to manually append the array I used `ls` to get the scripts them format it with `sed` you can take a look at my latest commit.